net/dns/manager_test.go

```diff
@@ -591,12 +591,36 @@ func TestManager(t *testing.T) {
 				DefaultResolvers: mustRes("https://dns.nextdns.io/c3a884"),
 			},
 			os: OSConfig{
-				Nameservers: mustIPs("100.100.100.100"),
+				Nameservers: mustIPs("fd60:627a:a42b::53"),
 			},
 			rs: resolver.Config{
 				Routes: upstreams(".", "https://dns.nextdns.io/c3a884"),
 			},
 		},
+		{
+			name: "coder",
+			in: Config{
+				OnlyIPv6: true,
+				Routes: map[dnsname.FQDN][]*dnstype.Resolver{
+					"coder.": mustRes("fd60:627a:a42b::53"),
```

Comment:
This isn't how we would configure things.  The serviceIP already is `fd60:627a:a42b`, so this would tell the resolver to use itself as an upstream --- e.g. a loop.

Reply:
Ah yep, I follow - this should just be `nil`